core/src/App.html

```diff
@@ -304,11 +318,27 @@
           }
         });
         window.addEventListener('message', async e => {
+          const allIframes = Iframe.getAllIframes(node);
+          const isOriginWindow = (iframe) => e.source === iframe.contentWindow;
+          const isOriginModal = () => this.get().modalIframe && e.source === this.get().modalIframe.contentWindow;
+          const getTargetIframe = (iframe) => {
+            return (isOriginWindow(iframe) || isOriginModal())
```

Comment:
This is not quite right, is it? It will return ANY 'iframe' if isOriginModal(). If isOriginModal then it should return the modal iframe.